Update README.md (#5583)
parse-community/parse-server: README.md

    - [Running elsewhere](#running-parse-server-elsewhere)
        - [Sample Application](#parse-server-sample-application)
        - [Parse Server + Express](#parse-server--express)
    - [Configuration](#configuration)
        - [Basic Options](#basic-options)
        - [Client Key Options](#client-key-options)
        - [Email Verification & Password Reset](#email-verification-and-password-reset)
        - [Using Environment Variables](#using-environment-variables-to-configure-parse-server)
        - [Available Adapters](#available-adapters)
        - [Configuring File Adapters](#configuring-file-adapters)
        - [Logging](#logging)

> - [Live Queries](#live-queries)

Reviewer: Seems like it would make more sense for this to be a subsection of `Configuration`?

Author: I though so as well. I figured because its a very important feature it needs it own section. Also, its a separate server (you could do startLiveQueryServer: true but that's undocumented).

Reviewer: fair point

`GridFSBucketAdapter` is used by default and requires no setup, but if you're interested in using S3 or Google Cloud Storage, additional configuration information is available in the [Parse Server guide](http://docs.parseplatform.org/parse-server/guide/#configuring-file-adapters).

### Logging

Parse Server will, by default, log:
* to the console
* daily rotating files as new line delimited JSON

> Logs are also be viewable in Parse Dashboard.

Reviewer: ```suggestion
Logs are also viewable in Parse Dashboard.
```

**Want to log each request and response?** Set the `VERBOSE` environment variable when starting `parse-server`. Usage :-  `VERBOSE='1' parse-server --appId APPLICATION_ID --masterKey MASTER_KEY`

> **Want logs to be in placed in other folder?** Pass the `PARSE_SERVER_LOGS_FOLDER` environment variable when starting `parse-server`. Usage :-  `PARSE_SERVER_LOGS_FOLDER='<path-to-logs-folder>' parse-server --appId APPLICATION_ID --masterKey MASTER_KEY`

Reviewer: ```suggestion
**Want logs to be in placed in a different folder?** Pass the `PARSE_SERVER_LOGS_FOLDER` environment variable when starting `parse-server`. Usage :-  `PARSE_SERVER_LOGS_FOLDER='<path-to-logs-folder>' parse-server --appId APPLICATION_ID --masterKey MASTER_KEY`
```

**Want to log specific levels?** Pass the `logLevel` parameter when starting `parse-server`. Usage :-  `parse-server --appId APPLICATION_ID --masterKey MASTER_KEY --logLevel LOG_LEVEL`

> **Want new line delimited JSON error logs (for consumption by CloudWatch, Google Cloud Logging, etc.)?** Pass the `JSON_LOGS` environment variable when starting `parse-server`. Usage :-  `JSON_LOGS='1' parse-server --appId APPLICATION_ID --masterKey MASTER_KEY`

Reviewer: ```suggestion
**Want new line delimited JSON error logs (for consumption by CloudWatch, Google Cloud Logging, etc)?** Pass the `JSON_LOGS` environment variable when starting `parse-server`. Usage :-  `JSON_LOGS='1' parse-server --appId APPLICATION_ID --masterKey MASTER_KEY`
```

# Live Queries

Live queries are meant to be used in real-time reactive applications, where just using the traditional query paradigm could cause several problems, like increased response time and high network and server usage. Live queries should be used in cases where you need to continuously update a page with fresh data coming from the database, which often happens in (but is not limited to) online games, messaging clients and shared to-do lists.

> Take a look at [Live Query Guide](https://docs.parseplatform.org/parse-server/guide/#live-queries), [Live Query Server Setup Guide](https://docs.parseplatform.org/parse-server/guide/#scalability) and [Live Query Protocol Specification](https://github.com/parse-community/parse-server/wiki/Parse-LiveQuery-Protocol-Specification). You can setup a standalone server or multiple instances for scalability (recommended).

Reviewer: > `[Live Query Server Setup Guide](https://docs.parseplatform.org/parse-server/guide/#scalability)`

Shouldn't the link here be "https://docs.parseplatform.org/parse-server/guide/#livequery-server"?

Author: I linked to the recommended setup. A lot of live query server complaints are because redisURL is needed for PubSub.

Reviewer: Got it 👍
